src/test/test-calendarspec.c

```diff
@@ -43,35 +44,30 @@ static void test_next(const char *input, const char *new_tz, usec_t after, usec_
         if (old_tz)
                 old_tz = strdupa(old_tz);
 
-        if (new_tz) {
-                char *colon_tz;
+        if (!isempty(new_tz))
+                new_tz = strjoina(":", new_tz);
 
-                colon_tz = strjoina(":", new_tz);
-                assert_se(setenv("TZ", colon_tz, 1) >= 0);
-        } else
-                assert_se(unsetenv("TZ") >= 0);
+        assert_se(set_unset_env("TZ", new_tz, true) == 0);
         tzset();
 
         assert_se(calendar_spec_from_string(input, &c) >= 0);
```

Comment:
CI is unhappy:

> Assertion 'calendar_spec_from_string(input, &c) >= 0' failed at src/test/test-calendarspec.c:53, function _test_next(). Aborting.

Reply:
Hmmmmm
```
line 216: "2017-09-09 20:42:00 Pacific/Auckland" new_tz=:EET
At: Sat 2017-09-09 11:42:00.000000 EEST
Assertion 'calendar_spec_from_string(input, &c) >= 0' failed at src/test/test-calendarspec.c:53, function _test_next(). Aborting.
```

Reply:
Duh, I forgot that we install all timezone files we need in the tests. Should be fixed now.